src/main/java/org/spin/grpc/service/field/payment/PaymentInfoService.java

```diff
@@ -0,0 +1,93 @@
+package org.spin.grpc.service.field.payment;
+
+import org.adempiere.exceptions.AdempiereException;
+// import org.adempiere.exceptions.AdempiereException;
+import org.compiere.util.CLogger;
+import org.spin.backend.grpc.common.ListLookupItemsResponse;
+// import org.spin.backend.grpc.common.ListLookupItemsResponse;
+import org.spin.backend.grpc.field.payment.ListPaymentInfoRequest;
+import org.spin.backend.grpc.field.payment.ListPaymentInfoResponse;
+import org.spin.backend.grpc.field.payment.ListBankAccountRequest;
+import org.spin.backend.grpc.field.payment.ListBusinessPartnersRequest;
+// import org.spin.backend.grpc.field.payment.ListPaymentInfoResponse;
```

Comment:
remove this comments

Reply:
https://github.com/solop-develop/adempiere-grpc-server/pull/781/commits/84677b582acd7aed13ceb674b8bd15361e8788e2